package.json

```diff
@@ -9,16 +9,21 @@
     "lint-fix": "standard --fix",
     "lint-commit": "test/lint-commits.sh",
     "check": "standard && npm run test-deps && npm run lint-commit",
-    "precoverage": "test/pretest.sh",
-    "coverage": "npm run check && nyc node test/test.js",
-    "postcoverage": "test/posttest.sh",
+    "coverage": "test ! -z $CI && npm run test-suite-coverage || COVERAGE=true ./test/script/docker/run_tests_dev.sh",
```

Comment:
I'd prefer if we could encode this sort of logic in the shell scripts rather than having this much complexity in npm scripts.

Reply:
I actually had this logic inside the script originally, but moved it out here because it felt wrong to run a script meant for development in CI, only to have it relay the call to the CI script and relay the exit code. But maybe I could make a "routing" script to figure out which sub-script to run based on the environment? Or I could just merge the two scripts together? What do you think?

Reply:
Fixed